content/millennium/r4/financial/general/charge-item.md

```diff
@@ -83,19 +83,22 @@ Search for ChargeItems that meet supplied query parameters:
 
 ### Parameters
 
- Name                  | Required?       | Type          | Description
------------------------|-----------------|---------------|-------------------------------------------------------
- `_id`                 | This or context | [`token`]     | The logical resource id associated with the ChargeItem.
- `context`             | This or _id     | [`token`]     | Encounter associated with event.
- `-status`             | See notes       | [`token`]     | The status of the ChargeItem. Example: `billable`
- [`_count`]            | No              | [`number`]    | The maximum number of results to return.
+ Name                  | Required?                  | Type          | Description
+-----------------------|----------------------------|---------------|-------------------------------------------------------
+ `_id`                 | This or context or account | [`token`]     | The logical resource id associated with the ChargeItem.
+ `context`             | This or _id or account     | [`reference`] | Encounter associated with event.
+ `account`             | This or context or _id     | [`reference`] | Account associated with the ChargeItem.
```

Comment:
nit: let's have parameters in the same order as context. i.e.
```ruby
This or _id or context
```

Reply:
Addressed in 2e4df3f24bdeb9538a6e6875435ea5f695bd59b5

Reply:
That commit doesn't address this.

Reply:
24bcb410398f2d17801fb6aec8534be947458102